src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let totalPrice = price * days;
+
+  if (days >= 7) {
+    totalPrice -= 50;
+  } else if (days >= 3 && days < 7) {
+    totalPrice -= 20;
+  }
```

Comment:
You can create separate if statement and return the correct totalPrice inside every if statements instead of else if structure.

Also move this 'magic numbers' 20 and 50 to variables

Reply:
done